tests/v2/validation/charts/installations_test.go

```diff
@@ -0,0 +1,227 @@
+		require.NoError(i.T(), err)
+
+		alertingChartInstallOption := &charts.InstallOptions{
+			Cluster:   i.cluster,
+			Version:   latestAlertingVersion,
+			ProjectID: i.project.ID,
+		}
+
+		alertingFeatureOption := &charts.RancherAlertingOpts{
+			SMS:   true,
+			Teams: true,
+		}
+
+		i.T().Logf("Installing alerting chart with the latest version in cluster [%v] with version [%v]", i.cluster.Name, latestAlertingVersion)
+		err = charts.InstallRancherAlertingChart(client, alertingChartInstallOption, alertingFeatureOption)
+		require.NoError(i.T(), err)
+	}
+
+	isUsingRegistry, err := registries.CheckAllClusterPodsForRegistryPrefix(client, i.cluster.ID, i.registrySetting.Value)
+	require.NoError(i.T(), err)
+	assert.Truef(i.T(), isUsingRegistry, "Checking if using correct registry prefix")
+}
+
+func (i *InstallationTestSuite) TestInstallLoggingChart() {
+	client, err := i.client.WithSession(i.session)
+	require.NoError(i.T(), err)
+
+	loggingChart, err := charts.GetChartStatus(client, i.project.ClusterID, charts.RancherLoggingNamespace, charts.RancherLoggingName)
+	require.NoError(i.T(), err)
+
+	if !loggingChart.IsAlreadyInstalled {
+		// Get latest versions of logging
+		latestLoggingVersion, err := client.Catalog.GetLatestChartVersion(charts.RancherLoggingName, catalog.RancherChartRepo)
+		require.NoError(i.T(), err)
+
+		loggingChartInstallOption := &charts.InstallOptions{
+			Cluster:   i.cluster,
+			Version:   latestLoggingVersion,
+			ProjectID: i.project.ID,
+		}
+
+		loggingChartFeatureOption := &charts.RancherLoggingOpts{
+			AdditionalLoggingSources: true,
+		}
+
+		i.T().Logf("Installing logging chart with the latest version in cluster [%v] with version [%v]", i.cluster.Name, latestLoggingVersion)
+		err = charts.InstallRancherLoggingChart(client, loggingChartInstallOption, loggingChartFeatureOption)
+		require.NoError(i.T(), err)
+	}
+
+	isUsingRegistry, err := registries.CheckAllClusterPodsForRegistryPrefix(client, i.cluster.ID, i.registrySetting.Value)
+	require.NoError(i.T(), err)
+	assert.Truef(i.T(), isUsingRegistry, "Checking if using correct registry prefix")
+}
+
+func (i *InstallationTestSuite) TestInstallIstioChart() {
+	i.TestInstallMonitoringChart()
+
+	client, err := i.client.WithSession(i.session)
+	require.NoError(i.T(), err)
+
+	istioChart, err := charts.GetChartStatus(client, i.project.ClusterID, charts.RancherIstioNamespace, charts.RancherIstioName)
+	require.NoError(i.T(), err)
+
+	if !istioChart.IsAlreadyInstalled {
+		// Get latest versions of logging
+		latestIstioVersion, err := client.Catalog.GetLatestChartVersion(charts.RancherIstioName, catalog.RancherChartRepo)
+
+		require.NoError(i.T(), err)
+
+		istioChartInstallOption := &charts.InstallOptions{
+			Cluster:   i.cluster,
+			Version:   latestIstioVersion,
+			ProjectID: i.project.ID,
+		}
+
+		istioChartFeatureOption := &charts.RancherIstioOpts{
```

Comment:
This looks good. One question about these options. If someone needed to modify this test, where would they look up what options are needed to be set?  Also, are these just enabling things or installing extra things? If so, it might be beneficial to name these with the action they are doing like `EnableIngressGateways`

Reply:
Makes sense